guardian/support-dotcom-components
# Support Abandoned Basket AB Test

## What does this change?

Change the abandonedBasketMatches function to also check the mvt value. If the user is in the control, do not show the abandoned basket banner. The mvt checking is repeated when setting the cookie in [support-frontend](https://github.com/guardian/support-frontend/pull/6086)



## How to test

Unit tests cover this, but to test manually - 

Banner not shown to control group:
1. ensure cookie is set
2. use MVT ID that implies control
3. `showMeTheBanner`
4. should not see an abandoned basket banner

Banner shown to variant group:
Repeat procedure but invert steps 2. and 4.
